### PR TITLE
feat: newsletter page

### LINKED ISSUE #657: Link Subscription Newsletter
## Description

We would like to create a link to subscribe to the Starknet.id newsletter.

### PR DESCRIPTION
**Pull request by Marchand-Nicolas**
close: #657

### CODE REVIEW

**Comment by Marchand-Nicolas** [comment]
API missing

**Comment by Marchand-Nicolas** [comment]
Needs this before merging: https://github.com/starknet-id/api.sales.starknet.id/pull/7

**Review by fricoben — CHANGES_REQUESTED** [review]
LGTM except some wording I changed on vercel

File: pages/newsletter.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+const NewsletterPage: NextPage = () => {
+  const { address } = useAccount();
+  const [error, setError] = useState<string>("");
+  const submitHandler = () => {
+    const email = (document.getElementById("email") as HTMLInputElement).value;
+    if (!email) {
+      setError("Please enter your email address.");
+      return;
+    }
+    fetch(`${process.env.NEXT_PUBLIC_SALES_SERVER_LINK}/newsletter_subscribe`, {
```
Comment (fricoben) [inline pages/newsletter.tsx:17]: Did you test that this back-end also triggers an add in the Mailerlite database ?

**Review by fricoben — APPROVED** [review]
Code Lgtm, to merge when when we'll test and back-end will be in prod

**Review by Kevils — CHANGES_REQUESTED** [review]
image
image

The email input, when focused, looks a bit strange. We could use the same background and text color as the domain form :)

**Review by fricoben — CHANGES_REQUESTED** [review]
I can subscribe even if my mail is not valid. 

We already have a function called `isValidEmail` in stringService for you to disable/enable the button when the email si valid.

**Review by fricoben — APPROVED** [review]
lgtm

**Pull request merged**